components/server/src/orgs/organization-service.spec.db.ts

```diff
@@ -162,30 +122,63 @@ describe("OrganizationService", async () => {
         owner = previouslyMember;
 
         // owner can downgrade themselves only if they are not the last owner
-        try {
-            await os.setOrganizationMemberRole(owner.id, org.id, owner.id, "member");
-            expect.fail("should have thrown");
-        } catch (error) {
-            expect(ApplicationError.hasErrorCode(error) && error.code, error.message).to.equal(ErrorCodes.CONFLICT);
-        }
+        await expectError(ErrorCodes.CONFLICT, () =>
+            os.setOrganizationMemberRole(owner.id, org.id, owner.id, "member"),
+        );
 
         // owner can delete themselves only if they are not the last owner
-        try {
-            await os.setOrganizationMemberRole(owner.id, org.id, owner.id, "member");
-            expect.fail("should have thrown");
-        } catch (error) {
-            expect(ApplicationError.hasErrorCode(error) && error.code).to.equal(ErrorCodes.CONFLICT);
-        }
+        await expectError(ErrorCodes.CONFLICT, () => os.removeOrganizationMember(owner.id, org.id, owner.id));
 
         // members can remove themselves
         await os.removeOrganizationMember(member.id, org.id, member.id);
 
-        try {
-            // try remove the member again
-            await os.removeOrganizationMember(member.id, org.id, member.id);
-            expect.fail("should have thrown");
-        } catch (e) {
-            expect(ApplicationError.hasErrorCode(e) && e.code).to.equal(ErrorCodes.NOT_FOUND);
-        }
+        // try remove the member again
+        await expectError(ErrorCodes.NOT_FOUND, () => os.removeOrganizationMember(member.id, org.id, member.id));
+    });
+
+    it("should listOrganizationsByMember", async () => {
+        await os.createOrganization(owner.id, "org1");
+        await os.createOrganization(owner.id, "org2");
+        let orgs = await os.listOrganizationsByMember(owner.id, owner.id);
+        expect(orgs.length).to.eq(3);
+        orgs = await os.listOrganizationsByMember(member.id, owner.id);
+        expect(orgs.length).to.eq(1);
+        orgs = await os.listOrganizationsByMember(stranger.id, owner.id);
```

Comment:
Ah, I would have expected a NOT_FOUND here. :thinking:

Reply:
Mis-read/-unstood what the method does. Still a tad strange... shouldn't this method only be callable on the user itself? Feels like I'm missing sth...